2023-02-13 16:28:35,543 INFO    MainThread:1620 [wandb_setup.py:_flush():68] Configure stats pid to 1620
2023-02-13 16:28:35,544 INFO    MainThread:1620 [wandb_setup.py:_flush():68] Loading settings from C:\Users\simplon\.config\wandb\settings
2023-02-13 16:28:35,544 INFO    MainThread:1620 [wandb_setup.py:_flush():68] Loading settings from C:\Users\simplon\Desktop\Notebooks\Dossier_App_Fil_Rouge\notebook\wandb\settings
2023-02-13 16:28:35,544 INFO    MainThread:1620 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-02-13 16:28:35,545 INFO    MainThread:1620 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-02-13 16:28:36,564 INFO    MainThread:1620 [wandb_init.py:_log_setup():478] Logging user logs to C:\Users\simplon\Desktop\Notebooks\Dossier_App_Fil_Rouge\notebook\wandb\run-20230213_162835-e5myir1e\logs\debug.log
2023-02-13 16:28:36,565 INFO    MainThread:1620 [wandb_init.py:_log_setup():479] Logging internal logs to C:\Users\simplon\Desktop\Notebooks\Dossier_App_Fil_Rouge\notebook\wandb\run-20230213_162835-e5myir1e\logs\debug-internal.log
2023-02-13 16:28:36,565 INFO    MainThread:1620 [wandb_init.py:_jupyter_setup():428] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000019BE69F4E20>
2023-02-13 16:28:36,567 INFO    MainThread:1620 [wandb_init.py:init():518] calling init triggers
2023-02-13 16:28:36,567 INFO    MainThread:1620 [wandb_init.py:init():521] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.02, 'architecture': 'ML', 'dataset': 'CIFAR-100', 'epochs': 10}
2023-02-13 16:28:36,567 INFO    MainThread:1620 [wandb_init.py:init():537] re-initializing run, found existing run on stack: qho6c62j
2023-02-13 16:28:36,571 INFO    MainThread:1620 [wandb_run.py:_finish():1844] finishing run youcef-b/<project_Loan_Prediction>/qho6c62j
2023-02-13 16:28:36,571 INFO    MainThread:1620 [wandb_run.py:_atexit_cleanup():2070] got exitcode: 0
2023-02-13 16:28:36,572 INFO    MainThread:1620 [wandb_run.py:_restore():2053] restore
2023-02-13 16:28:36,572 INFO    MainThread:1620 [wandb_run.py:_restore():2059] restore done
2023-02-13 16:28:41,080 INFO    MainThread:1620 [wandb_run.py:_footer_history_summary_info():3427] rendering history
2023-02-13 16:28:41,081 INFO    MainThread:1620 [wandb_run.py:_footer_history_summary_info():3459] rendering summary
2023-02-13 16:28:41,102 INFO    MainThread:1620 [wandb_run.py:_footer_sync_info():3383] logging synced files
2023-02-13 16:28:41,130 INFO    MainThread:1620 [wandb_init.py:init():571] starting backend
2023-02-13 16:28:41,130 INFO    MainThread:1620 [wandb_init.py:init():575] setting up manager
2023-02-13 16:28:41,140 INFO    MainThread:1620 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-02-13 16:28:41,150 INFO    MainThread:1620 [wandb_init.py:init():582] backend started and connected
2023-02-13 16:28:41,233 INFO    MainThread:1620 [wandb_run.py:_label_probe_notebook():1202] probe notebook
2023-02-13 16:28:41,235 INFO    MainThread:1620 [wandb_run.py:_label_probe_notebook():1212] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-02-13 16:28:41,235 INFO    MainThread:1620 [wandb_init.py:init():670] updated telemetry
2023-02-13 16:28:41,326 INFO    MainThread:1620 [wandb_init.py:init():710] communicating run to backend with 60.0 second timeout
2023-02-13 16:28:42,805 INFO    MainThread:1620 [wandb_run.py:_on_init():2121] communicating current version
2023-02-13 16:28:43,121 INFO    MainThread:1620 [wandb_run.py:_on_init():2125] got version response upgrade_message: "wandb version 0.13.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-02-13 16:28:43,121 INFO    MainThread:1620 [wandb_init.py:init():758] starting run threads in backend
2023-02-13 16:28:59,515 INFO    MainThread:1620 [wandb_run.py:_console_start():2101] atexit reg
2023-02-13 16:28:59,515 INFO    MainThread:1620 [wandb_run.py:_redirect():1959] redirect: SettingsConsole.WRAP_RAW
2023-02-13 16:28:59,515 INFO    MainThread:1620 [wandb_run.py:_redirect():2024] Wrapping output streams.
2023-02-13 16:28:59,515 INFO    MainThread:1620 [wandb_run.py:_redirect():2046] Redirects installed.
2023-02-13 16:28:59,517 INFO    MainThread:1620 [wandb_init.py:init():798] run started, returning control to user process
2023-02-13 16:28:59,521 INFO    MainThread:1620 [wandb_run.py:_finish():1844] finishing run youcef-b/my-awesome-project/e5myir1e
2023-02-13 16:28:59,522 INFO    MainThread:1620 [jupyter.py:save_history():449] not saving jupyter history
2023-02-13 16:28:59,522 INFO    MainThread:1620 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-02-13 16:28:59,522 INFO    MainThread:1620 [wandb_init.py:_jupyter_teardown():410] cleaning up jupyter logic
2023-02-13 16:28:59,523 INFO    MainThread:1620 [wandb_run.py:_atexit_cleanup():2070] got exitcode: 0
2023-02-13 16:28:59,524 INFO    MainThread:1620 [wandb_run.py:_restore():2053] restore
2023-02-13 16:28:59,524 INFO    MainThread:1620 [wandb_run.py:_restore():2059] restore done
2023-02-13 16:29:04,559 INFO    MainThread:1620 [wandb_run.py:_footer_history_summary_info():3427] rendering history
2023-02-13 16:29:04,560 INFO    MainThread:1620 [wandb_run.py:_footer_history_summary_info():3459] rendering summary
2023-02-13 16:29:04,584 INFO    MainThread:1620 [wandb_run.py:_footer_sync_info():3383] logging synced files
